src/main/scala/com/typesafe/sbt/packager/archetypes/systemloader/package.scala

```diff
@@ -71,5 +71,5 @@ package object systemloader {
     loader.toString + "/" + name
 
   private def overrideFromFile(sourceDirectory: File, loader: ServerLoader, name: String): Option[URL] =
-    Option(sourceDirectory / "templates" / "systemloader" / loader.toString).filter(_.exists).map(_.toURI.toURL)
+    Option(sourceDirectory / "templates" / "systemloader" / loader.toString / name).filter(_.exists).map(_.toURI.toURL)
```

Comment:
Thanks for fixing this. This quite some bug. Along with 6 tests that tested the wrong implementation:

```
[error] (*:scripted) Failed tests:
[error] 	rpm / override-start-script-systemd
[error] 	rpm / override-start-script-systemv
[error] 	rpm / override-start-script-upstart
[error] 	debian / override-start-script-systemd
[error] 	debian / override-start-script-systemv
[error] 	debian / override-start-script-upstart
```